### PR TITLE
Mylast

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/api.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 @bp.route('/test')
 def test():
-    return 'testo'
+    return 'testoz'
```
Comment (bplusv) [inline src/api.py:16]: lgtm

**Review by bplusv — CHANGES_REQUESTED** [review]
lgtm1

File: src/api.py (hunk 1 above)
Comment (ghost) [reply src/api.py:16]: ok @bplusv  let me check

Comment (bplusv) [reply src/api.py:16]: ok @windwaker64  let go

**Review by bplusv — APPROVED** [review]

**Pull request merged**